src/rpc/rawtransaction.cpp

```diff
@@ -3382,7 +3385,11 @@ static const CRPCCommand commands[] =
 { //  category              actor (function)            argNames
   //  --------------------- ------------------------        -----------------------     ----------
     { "rawtransactions",    &getrawtransaction,           },
+#ifdef __SEQUENTIA__
```

Comment:
I would avoid this kind of intermittent line modifications, but put the whole function in a `__SEQUENTIA__` switch and use one for Sequentia and the other if it is regular Elements. This way if they introduce some new rpc in a new version it the diff won't conflict with our changes as it will be in a separate block.